Декоративные элементы на главной странице

AlexPolyakovWeb opened the pull request:
Не разобрался как убрать чёрный бордер при фокусе, как сделать подчёркивание активного меню

---
:mortar_board: [Декоративные элементы на главной странице](https://up.htmlacademy.ru/htmlcss/33/user/1967959/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/1967959-sedona-33/8/

AlexPolyakovWeb committed:
Декоротивные главной + состояния

Не разобрался как убрать чёрный бордер при фокусе, как сделать подчёркивание активного меню

AlexPolyakovWeb committed:
Сделал обводку в активном состоянии

4err commented:
https://take.ms/yfnXo Какой-то отступ и тень. 
В главном меню потерялась зеленая иконка у избранного. 
Логотип не по макету, обрати внимание, что он наползает на нижний блок. 
При фокусе в инпут "Подписаться" текст плейсхолдера прыгает. Этого не должно происходить.

Обводка при фокусе это outline, а не бордер, поэтому и не убирается :)
Нижнее подчеркивание нужно делать доп классом и псевдоэлементом.

AlexPolyakovWeb committed:
Внёс правки

Добавил в навигации недостающий элемент, поправил фон в блоке подписки, подправил инпут при фокусе

AlexPolyakovWeb committed:
Сделал подчёркивания пунктов меню

AlexPolyakovWeb committed:
Сверстал каталог
